doc/whats_new.rst

```diff
@@ -837,7 +844,64 @@ Miscellaneous
   ensemble estimators (deriving from :class:`ensemble.BaseEnsemble`)
   now only have ``self.estimators_`` available after ``fit``.
   :issue:`7464` by `Lars Buitinck`_ and `Loic Esteve`_.
-
+   - Deprecate the ``y`` parameter in ``transform`` and ``inverse_transform``.
+     The method  should not accept ``y`` parameter, as it's used at the prediction time.
+     :issue:`8174` by :user:`Tahar Zanouda <tzano>`, `Alexandre Gramfort`_
+     and `Raghav RV`_.
+
+   - SciPy >= 0.13.3 and NumPy >= 1.8.2 are now the minimum supported versions
```

Comment:
It looks like there is some unwanted changes in this diff. The only thing that should be added is your paragraph about `store_covariances_` and `covariances_`